java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S3751_java.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "\"@RequestMapping\" methods should not be \"private\"",
+  "title": "\"@RequestMapping\" methods should be \"public\"",
```

Comment:
This should not be reverted, the related PR is not yet merged! Can you review it?https://github.com/SonarSource/rspec/pull/584

Reply:
good catch!
```suggestion
  "title": "\"@RequestMapping\" methods should not be \"private\"",
```